Description: Upstream changes introduced in version 0.1-0ubuntu1~ppa1n
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 lg4l (0.1-0ubuntu1~ppa1n) natty; urgency=low
 .
   * Initial release
 .
 The person named in the Author field signed this changelog entry.
Author: Brett Smith <tanktarta@blueyonder.co.uk>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- lg4l-0.1.orig/Makefile
+++ lg4l-0.1/Makefile
@@ -10,7 +10,7 @@ else
 
 KVERSION = $(shell uname -r)
 KDIR := /lib/modules/$(KVERSION)/build
-MODULE_INSTALL_DIR := /lib/modules/$(KVERSION)/updates/g-series
+MODULE_INSTALL_DIR := $(DESTDIR)/updates/g-series
 MODS := hid-g13.ko hid-g15.ko hid-g19.ko hid-gfb.ko hid-g110.ko
 PWD := $(shell pwd)
 
